Fix incorrect use of dt_print in email export target

victoryforce opened the pull request:
We're forming a string to print here with multiple function calls, so the dt_print function isn't appropriate here because it adds a timestamp to each segment of the output string.

ralfbrown commented:
An alternative if keeping the timestamp is desireable would be using `g_strconcat` or the like to build a single string from the arglist so that it can be written with a single call to `dt_print`.

victoryforce commented:
I agree with you, keeping the timestamp is desirable, at least from the aesthetic point of view for the sake of uniformity of the log. Changed. Thank you!

Also replaced DT_DEBUG_ALWAYS with DT_DEBUG_IMAGEIO as this is only a debug print and not a critical error message.

TurboGit reviewed (approved):
Thanks!
